addon/plugins/structure-plugin/node.ts

```diff
@@ -51,7 +52,14 @@ export type StructureType =
   | 'subsection'
   | 'paragraph';
 
-export const emberNodeConfig: () => EmberNodeConfig = () => {
+type StructureConfig = {
+  fullLenghtArticles: boolean;
+  onlyArticleSpecialName: boolean;
```

Comment:
As the `structureConfig` is an optional argument, I would also make both of these fields individually optional.

Reply:
good catch :)